### PR TITLE
ETCD-199: bump etcd v3.4.16 

### PR DESCRIPTION
**Pull request by hexfusion**
This PR bumps etcd to v3.4.16 from v3.4.14 using golang  1.12 and is part of a multi PR migration[1] to get to etcd 3.5.0.

[1]  https://github.com/openshift/etcd/issues/85

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.12 AS builder
```
Comment (lilic) [inline Dockerfile.rhel:1]: Should we switch to 1.15 or do you want to do that in separate PR?

**Review by lilic — COMMENTED** [review]
Nice thanks! 🎉

**Comment by lilic** [comment]
Reported the strange tide msg to dpp team.

**Comment by lilic** [comment]
We need to wait for https://github.com/openshift/etcd/pull/84/files PR.

File: Dockerfile.rhel (hunk 1 above)
Comment (hexfusion) [reply Dockerfile.rhel:1]: yeah lets merge this as 1.12 them pivot to go1.15

Comment (hexfusion) [reply Dockerfile.rhel:1]: well on second thought I will change this then pivot in release.

**Comment by hexfusion** [comment]
infra route53

/test configmap-scale

**Comment by hexfusion** [comment]
> failed to acquire lease for "aws-quota-slice": resources not found

/test e2e-aws-upgrade

**Comment by hexfusion** [comment]
> failed to acquire lease for "aws-quota-slice": resources not found

/test e2e-aws-serial

**Comment by lilic** [comment]
Sadly there is an AWS limit we reached, they are still working on it, so not sure when this will pass. Maybe worth waiting until Monday to retest again?

**Review by lilic — APPROVED** [review]
/lgtm
/approve 

🎉

**Comment by hexfusion** [comment]
route53 ...
/test configmap-scale

**Comment by hexfusion** [comment]
infra ..............

/test configmap-scale
